app/components/trainees/confirmation/contact_details/view.rb

```diff
@@ -0,0 +1,54 @@
+module Trainees
+  module Confirmation
+    module ContactDetails
+      class View < GovukComponent::Base
+        include SanitizeHelper
+
+        attr_accessor :trainee
+
+        def initialize(trainee:)
+          @trainee = trainee
+          @not_provided_copy = I18n.t("components.confirmation.not_provided")
+        end
+
+        def address
+          return @not_provided_copy if trainee.locale_code.nil?
```

Comment:
![image](https://user-images.githubusercontent.com/470137/95980361-f5bef780-0e14-11eb-88be-3560f4ff075c.png)

You can still end up with that by selecting you from uk or non uk and not setting any address lines...